index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Minesweep</title>
+    <link rel="stylesheet" href="style.css" />
+    <script type="module" src="script.js"></script>
+</head>
+
+<body>
+    <header>
+        <h1>Minesweep</h1>
+    </header>
+    <div></div>
+    <main>
+        <article>
+            <div class="gameoptions">
```

Comment:
this may be an outdated naming convention, so I'll leave it up to you to confirm: css classes should be hyphen-separated, i.e. `btn-lg`, `btn-sm` or `grid-container` etc
```suggestion
            <div class="game-options">
```

Reply:
(same applies to all your css class naming, if applicable)

Reply:
good to know